sources/functions/utils

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+function dist_info() {
+	DIST_CODENAME="$(source /etc/os-release && echo "$VERSION_CODENAME")"
```

Comment:
Couple things here:

- I'm not 100% certain it makes sense to have this as a separate sub-function of utils. Could we not just have the variables defined if utils is imported?
- Do we need to rename variables?

Reply:
it should be moved out of utils altogether then and set at the earliest opportunity as exported variables and not a function. Where would that be?

I had just `id="$(source /etc/os-release && echo "$ID")"` and i think this conflicts with certain scripts.

Maybe it's not so great to export as the same or uppercase so.

```
os_codename="$(source /etc/os-release && echo "$VERSION_CODENAME")"
os_id="$(source /etc/os-release && echo "$ID")"
```

Reply:
Hmm, well I've already thought that we should have a case for things like `apt` functions which are sourced globally through `box`. One function to source all our global depends (`echos`, `apt`, `dist_info`) which we can also just source in setup after the repo has been cloned locally.

I guess `ID` is maybe not the *most* unique variable. Easier to choose unique globals than worry about clobbering simple named ones in app scripts.

Reply:
Since it's a file we are sourcing and there is no automation of these loaded as system vars, i don't think renaming them to something unique will do any harm. 

Really, if you wanted that, you could just source the file itself in box.

```bash
source /etc/os-release
```

But for the 2 vars we require i can't see it being a problem as long as the naming is predictable.

Maybe it should just go in box as.

```bash
export os_codename="$(source /etc/os-release && echo "$VERSION_CODENAME")"
export os_id="$(source /etc/os-release && echo "$ID")"
```